src/main.rs

```diff
@@ -30,17 +32,16 @@ pub mod process;
 use clap::{App, Arg, SubCommand};
 use config::ForeverConfig;
 use process::Processes;
-use std::env;
+
 use std::thread;
 use std::time;
+use util::set_panic_handler;
 
 include!(concat!(env!("OUT_DIR"), "/build_info.rs"));
 
 fn main() {
-    // Always print backtrace on panic.
-    env::set_var("RUST_BACKTRACE", "full");
-
-    logger::init();
+    micro_service_init!("cita-forever", "CITA:forever");
```

Comment:
You can't use this macro to do log init things！
This macro will modify the default panic behavior of rust, if a thread panic will cause the entire process to exit.
What is needed for forever is that thread panic doesn't matter, it will restart a thread to work again (check if other microservices are working properly), forever needs to run robustly.

Reply:
good point. here is the source code of `micro_service_init!`
https://github.com/cryptape/cita-common/blob/develop/util/src/init.rs#L5:L18